hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/strategy/PartitionAwareClusteringPlanStrategy.java

```diff
@@ -68,6 +75,7 @@ protected Stream<HoodieClusteringGroup> buildClusteringGroupsForPartition(String
             - (o1.getBaseFile().isPresent() ? o1.getBaseFile().get().getFileSize() : writeConfig.getParquetMaxFileSize())));
 
     long totalSizeSoFar = 0;
+    boolean partialScheduled = true;
```

Comment:
The default value should be false I guess.

Reply:
exactly Missing here. changed